src/library/scala/App.scala

```diff
@@ -37,6 +37,17 @@ import scala.collection.mutable.ListBuffer
  *  before the main method has been executed.'''''
  *
  *  Future versions of this trait will no longer extend `DelayedInit`.
+ * 
+ *  In Scala 3, `App` exists only in a limited form that does not support command line
```

Comment:
this makes it sound like the main and perhaps only limitation is the lack of support for command line arguments. but it's equally important that Scala 3 does not implement `DelayedInit`